### PR TITLE
Fix/cargo clippy

### PR DESCRIPTION
**Pull request by fvaleye**
# Description
Fix clippy errors

### CODE REVIEW

File: rust/src/storage/file/mod.rs [hunk 1]
```diff
@@ -148,7 +148,8 @@ mod tests {
             .put_obj(new_file_path.to_str().unwrap(), b"hello")
             .await
         {
-            panic!(format!("Expect put_obj to return Ok, got Err: {:#?}", e));
+            let error_message = format!("Expect put_obj to return Ok, got Err: {:#?}", e);
+            panic!("{}", error_message);
```
Comment (houqp) [inline rust/src/storage/file/mod.rs:152]: would be better to rewrite it as `panic!("Expect put_obj to return Ok, got Err: {:#?}", e)`

File: rust/src/storage/file/rename.rs [hunk 2]
```diff
@@ -119,8 +119,11 @@ mod tests {
                     }
                 }
             }
-            Err(e) => panic!(format!("expect std::io::Error, got: {:#}", e)),
-            Ok(()) => panic!("expect rename to fail with Err, but got Ok"),
+            Err(e) => {
+                let error_message = format!("expect std::io::Error, got: {:#}", e);
+                panic!("{}", error_message)
```
Comment (houqp) [inline rust/src/storage/file/rename.rs:124]: same here, we can just use `panic!` like `format!`.

File: rust/src/storage/file/mod.rs (hunk 1 above)
Comment (fvaleye) [reply rust/src/storage/file/mod.rs:152]: 👍

File: rust/src/storage/file/rename.rs (hunk 2 above)
Comment (fvaleye) [reply rust/src/storage/file/rename.rs:124]: 👍

**Comment by fvaleye** [comment]
@houqp wait for the CI but it is fixed for the error in Windows build (wrong space at the end of the filename 🤦 )

**Review by houqp — APPROVED** [review]
the windows build error looks weird, what's up with that?

**Comment by fvaleye** [comment]
> the windows build error looks weird, what's up with that?

The file `config.yml ` in ISSUE_TEMPLATE merged in `main` contains a space at the end.
It is fixed in this PR but I think that the CI failed because it tries to fetch the current main branch...

**Comment by houqp** [comment]
i see, that makes sense

**Comment by houqp** [comment]
@fvaleye Looks like the windows build is still failing on master, so it's something different.

**Comment by houqp** [comment]
oh, guess we need to delete the invalid config.yml file as well?

**Comment by fvaleye** [comment]
You are right! I fixed it 😃

**Pull request merged**